lib/data/remote/api_service.dart

```diff
@@ -1,12 +1,12 @@
-import 'package:c2s/data/get_entry_response_data.dart';
-import 'package:c2s/data/post_entries_request_data.dart';
-import 'package:c2s/data/entries_response_data.dart';
-import 'package:c2s/data/logout_response_data.dart';
-import 'package:c2s/data/post_entries_response_data.dart';
-import 'package:c2s/data/user_response_data.dart';
+import 'package:c2s/data/json_data/get_entry_response_data.dart';
```

Comment:
Why are you giving a constant value to the "createdBy" parameter in "getEntries" endpoint

Reply:
the part where it says createdby=all?